New r plots

DidDrog11 opened the pull request.

TimTaylor committed:
update data cleaning for new ccgs

DidDrog11 committed:
new shapefile and mapping

DidDrog11 committed:
new r plots

TimTaylor committed:
fix typo in lookup

TimTaylor commented on analyses-ccg.Rmd:
```diff
@@ -298,27 +298,35 @@ r_ccg <- counts_ccg_all %>%
   growth_rate() %>%
   rename("ccg" = "ccg_name")
 
+r_plot <- counts_ccg_all %>% fit_curve(model = "poisson") %>%
```
These plots do not correspond to the calculated r-values (i.e. they use `keep_last(3*7)`)

DidDrog11 replied:
Does keep_last work here? I think I tried that before and I was struggling to get a plot.

TimTaylor replied:
possibly not - it may need doing manually.  The key thing is the plots need to match with the number of days we've fitted the growth rate to.

TimTaylor replied:
actually - it should work there.  Let me know if there's an issue and we can dig in to further

TimTaylor reviewed (commented):
The map plot seems to have blown up in size (relative to the March maps).  Currently it is close to 150mb for a single page! We need to work out what the issue is.  I note that we are now using leaflet rather than mapview so that may be the issue. Other possible reasons are the complexity of plot (less likely) and potentially the shapefiles.

TimTaylor committed:
use reduced resolution ccg shapefiles

available from https://geoportal.statistics.gov.uk/datasets/d6acd30ad71f4e14b4de808e58d9bc4c_0?geometry=-23.081%2C50.530%2C18.799%2C55.169

DidDrog11 committed:
modify r plots, change colour limits r plot

DidDrog11 committed:
change colour ramp

TimTaylor commented:
LGTM - @thibautjombart do you want to take a look prior to merging.

thibautjombart commented:
> LGTM - @thibautjombart do you want to take a look prior to merging.

Testing it locally and will merge once it passes.

thibautjombart commented:
New results look cool, but there are a few glitches when compiling the CCG analyses locally for me.
1. The color scale for r values seems off, and ends up with grey for high values
![image](https://user-images.githubusercontent.com/10757154/114011189-3bf59080-9865-11eb-918f-0520ccc3759b.png)

2. Some 'new' CCGs seem no to have been matched against old ones and come up as increases:
![image](https://user-images.githubusercontent.com/10757154/114011296-5d567c80-9865-11eb-9307-3f14036c23d8.png)

TimTaylor commented:
Have you rerun the data cleaning script locally.  My guess is you have old data?
